charter.md

```diff
@@ -0,0 +1,336 @@
+  - Thanos is discussing adding one too [here](https://github.com/thanos-io/thanos/issues/4606).
+
+### Design: observe.io
+
+- visualization / details showing the actual request, and the infra that's generating the response. Useful as a doorway to engage with persons that find their way to one of the 2 main front doors (observe-k8s.dev, observe-k8s.io).
+
+          "Welcome! This is you and here is the web server that rendered the site you're currently observing :)  \
+
+  \
+  Enter! See what else is running!" \
+
+- For 2d (flat browser) and 3d (AR, MR, VR, 3d rendering in-browser) the site would show other CNCF Projects running, hosted by the CNCF Lab ([https://github.com/cncf/cluster](https://github.com/cncf/cluster))
+- Suggest [https://angular.io](angular.io)
+  - extensibility via drop-in Angular Controls
+  - modelling connections to back end API’s ([Services](https://angular.io/guide/architecture-services))
+  - shared data model enabling n views of same underlying data simply.
+
+### Design: observe.dev
+
+- Targeting developer persona
+
+###
+
+### CNCF interested parties and collaboration opportunities:
+
+#### SIG Instrumentation
+
+- Source of upstream k8s signals
+
+#### CNCF TAG App Deploy: GitOps WG
+
+- Open Standard for data format for reporting
+
+\
+TAG Observability.
+
+- How to visualize execution of Operators, Controllers en masse / aggregate.
+- Engage with SIG Instrumentation re: gaps and opportunities for signals from k8s
+- Collect and Curate templates, recipes, examples, demos highlighting the observation of cloud native systems and workloads (CNCF Projects).
+
+## Personas
+
+Gibbs Cullen from Chronosphere has been working on Personas, TODO ping to see if we can use these here.
+
+### Practitioner
+
+### Developer
+
+### Data Scientist / Analyst
+
+### Business Function Owner
+
+## Technology Stack (brainstorm)
+
+_Proposed: Brainstorming - this is not authoritative!_
+
+### Architecture
+
+TODO: 10k' Diagram
+
+### GitOps for $allTheThings
+
+- Flux v2 - base / common cluster services, workloads as desired
+- ArgoCD - workloads
+- …?
+
+### Platform Composition & Abstraction
+
+Use [crossplane.io](https://crossplane.io/) for the base kubernetes platform to abstract:
+
+- public cloud { AWS, Azure, GCP, … } where observe-k8s is running
+- Enable local development (kind, k3d, …) and iteration
+- CI, running github actions (kind)
+
+### Ingress: NGINX
+
+- It's portable, runs equally well locally (on a laptop in a kind cluster) or in public cloud
+
+### Prometheus operator
+
+- Provide cluster-local prometheus, alertmanager, …
+
+### Service Mesh: Linkerd
+
+- CNCF Graduated Project
+
+##
+
+## workloads: Objects of Observation (wOoO)
+
+[https://landscape.cncf.io/card-mode](https://landscape.cncf.io/card-mode), as of 2021-11-22.
+
+### CNCF Projects: Graduated (16)
+
+todo image
+### Incubating (25)
+
+todo image
+
+### CNCF Projects: Sandbox (66)
+
+todo image
+
+### Potential example workloads
```

Comment:
That section should be more prominent, there was consensus on the potential workloads candidate and this is kind of core.

Reply:
agreed and done